src/classes/BDI_DataImportService.cls

```diff
@@ -48,10 +50,27 @@ global with sharing class BDI_DataImportService {
     public BDI_DataImportService(Boolean isDryRun, BDI_FieldMapping fieldMapper) {
         contactService = new BDI_ContactService(this);
         this.isDryRun = isDryRun;
-
         this.fieldMapper = fieldMapper;
     }
 
+    /*******************************************************************************************************
+    * @description queries the Data Import Settings custom setting to determine what type of field mapping to
+    * use.
+    */
+    public static BDI_FieldMapping getDefaultFieldMapping(){
+        System.debug('BDI_DataImportService getDefaultFieldMapping');
```

Comment:
Hey @callen-sfdo - we should probably remove these debug statements prior to merging. Couple others below as well.

Reply:
I agree in general, but since this is not going into prod I was planning to leave them in until I finish the next ticket since I would probably end up putting a lot of them back in anyway.  Is that cool?